cnf-certification-test/preflight/suite.go

```diff
@@ -36,6 +36,12 @@ var _ = ginkgo.Describe(common.PreflightTestKey, func() {
 	})
 	ginkgo.ReportAfterEach(results.RecordResult)
 
+	// Add safeguard against running the tests if the docker config doesn't exist.
+	if env.GetDockerConfigFile() == "" {
```

Comment:
Shouldn't we also remove the `required:"true"` setting of the env var in the TestParameters struct (configuration.go)?
```
	PfltDockerconfig       string `split_words:"true" required:"true" envconfig:"PFLT_DOCKERCONFIG"`
```

Reply:
Yes, it's still failing when the `PFLT_DOCKERCONFIG` env var is unset:

time="2023-02-16T11:07:58+01:00" level=fatal msg="required key PFLT_DOCKERCONFIG missing value"

Reply:
Ah whoops, yup I need to remove that as well good catch.